app/gosns/delete_topic.go

```diff
@@ -0,0 +1,47 @@
+package gosns
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/Admiral-Piett/goaws/app"
+	"github.com/Admiral-Piett/goaws/app/common"
+	"github.com/Admiral-Piett/goaws/app/interfaces"
+	"github.com/Admiral-Piett/goaws/app/models"
+	"github.com/Admiral-Piett/goaws/app/utils"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func DeleteTopicV1(req *http.Request) (int, interfaces.AbstractResponseBody) {
+	requestBody := models.NewDeleteTopicRequest()
+	ok := utils.REQUEST_TRANSFORMER(requestBody, req, false)
+	if !ok {
+		log.Error("Invalid Request - DeleteTopicV1")
+		return utils.CreateErrorResponseV1("InvalidParameterValue", false)
+	}
+
+	topicArn := requestBody.TopicArn
+	uriSegments := strings.Split(topicArn, ":")
+	topicName := uriSegments[len(uriSegments)-1]
+
+	log.Println("Delete Topic - TopicName:", topicName)

```

Comment:
Can we make these all info/warning/debug/etc., whatever seems appropriate where we see them?  the `Print*` stuff doesn't abide by the set levels and is difficult to manage.

This one seems like `info` to me, but I don't feel strongly.